docs/content/docs/reference/nextjs/index.mdx

```diff
@@ -2,7 +2,7 @@
 title: 'Introduction'
 ---
 
-It is possible to use [`@nhost/react`](/reference/react) in any Next.js page that would be configured to render on the client-side.
+All the React hooks and helpers from [`@nhost/react`](/reference/react) are available in NextJs and are exported in the `@nhost/nextjs` package.
```

Comment:
Can we please convert `NextJs` to `Next.js` here?

Reply:
yup